plugin.js

```diff
@@ -1,36 +1,70 @@
-CKEDITOR.plugins.add( 'bylawlist', {
-    onLoad: function() {
-      CKEDITOR.addCss(
-        'ol.bylawlist,ol.bylawlist>li,ol.bylawlist>li ol>li,ol.bylawlist>li ol>li ol>li, ol.bylawlist>li ol>li ol>li ol>li{list-style-type:none;list-style-position:inside}ol.bylawlist{padding-left:1.5em}ol.bylawlist>li{counter-increment:first}ol.bylawlist>li:before{content:"(" counter(first,decimal) ") "}ol.bylawlist>li ol>li{counter-increment:second}ol.bylawlist>li ol>li:before{content:"(" counter(second,lower-alpha) ") "}ol.bylawlist>li ol>li ol>li{counter-increment:third}ol.bylawlist>li ol>li ol>li:before{content:"(" counter(third,lower-roman) ") "}ol.bylawlist>li ol>li ol>li ol>li{counter-increment:fourth}ol.bylawlist>li ol>li ol>li ol>li:before{content:"(" counter(fourth, decimal) "." counter(first, decimal) ") "}'
-      );
-    },
-    icons: 'bylawlist', // Bylaw List icon
-    init: function( editor ) {
-        editor.addCommand( 'insertBylawlist', {
-            exec: function( editor ) {
-              if ('li' == editor.getSelection().getStartElement().getName()) {  
-                if ('OL' == editor.getSelection().getStartElement().$.parentNode.nodeName) {
-                  var bylaw = editor.getSelection().getStartElement().$.parentNode;
-                  if(bylaw.className.indexOf("bylawlist") < 0){
-                     bylaw.className += "bylawlist";
-                  } 
-                  else {
-                    bylaw.className = "";
-                  }
-                  
-                }              
-              }
+CKEDITOR.plugins.add('bylawlist', {
+  onLoad: function () {
+    CKEDITOR.addCss(
+      'ol.bylawlist{list-style-type:none;list-style-position:inside;padding-left:1.5em}ol.bylawlist>li>ol,ol.bylawlist>li>ol>li>ol{list-style-type:inherit}ol.bylawlist li{counter-increment:bylawlist-counter}ol.bylawlist li:first-child{counter-reset:bylawlist-counter}ol.bylawlist>li:before{content:"(" counter(bylawlist-counter,decimal) ") "}ol.bylawlist>li>ol>li:before{content:"(" counter(bylawlist-counter,lower-alpha) ") "}ol.bylawlist>li>ol>li>ol>li:before{content:"(" counter(bylawlist-counter,lower-roman) ") "}ol.bylawlist>li>ol>li>ol>li>ol{list-style-type:decimal}ol.bylawlist>li>ol>li>ol>li>ol>li:before{content:none}'
```

Comment:
This is the compiled copy from the theme for the fourth level bylawlist

Reply:
Gross

Reply:
It’s just the css from _list.scss in the theme. So the lists show up in the editor as they would on display. You don’t actually edit this you just copy / paste the compiled version from Ottawa bootstrap.